apps/mobile-alphabet-e2e/src/app.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { device, element, by, expect } from 'detox';
+
+describe('MobileAlphabet', () => {
```

Comment:
I notice `detox` has test helpers as well.

Reply:
One thing you'll have to establish is a strategy for testing. When will you use Jest vs. something like Cypress \ automated client tests?